### DISCUSSION TITLE
How will the rise of AI affect drug development?

### DISCUSSION THREAD

**Discussion by harveyl888** [rinpharma/positconf-roundtables-2023#11] [Ideas]
We've already seen adoption of AI in drug discovery, but less so in drug development.  

- How will AI affect drug development over the next 5 years?
- What open source methods are available and will the industry move to develop tools using open source or adopt vendor proprietary technology?
- Is there a space for collaborative work or will the types of problems that AI can answer dictate a closed model?

**Comment by kkmann** [comment]
- effective use of LLMs to support coding in a clinical reporting evironment
- AI generated code and quality control

**Comment by bargop** [comment]
I think this is a huge topic. I know there are many pharma companies looking into this for things like clinical site location. I spoke with people from the FDA recently and asked what was keeping them awake at night. One person said AI and how they will define regulations. There are many pharma and biotech companies hiring in this space.

**Reply by kamilsi** [reply]
@bargop, your mention of AI in clinical site location resonates with me. I once spearheaded a project focused on optimizing clinical trial sites based on specific KPIs, including execution time. While the AI models were effective at identifying optimal locations to meet these objectives, it always led me to question the broader implications.

Choosing sites purely for speed or cost-efficiency raises important questions about the reliability and representativeness of trial results. Do we risk compromising the robustness of our data for the sake of expediency? Is the "optimized" location necessarily the best for capturing a diverse patient population, or could it skew results in some way?

I believe as AI becomes more integrated into drug development processes, we will also need to refine our KPIs and consider multi-dimensional optimization to ensure we are aligning with not just business goals but also scientific rigor and ethical considerations.

**Reply by kamilsi** [reply]
also I'd like to follow up the topic of regulations, especially in light of the recently adopted EU AI Act (you can read more about it [here](https://www.europarl.europa.eu/news/en/headlines/society/20230601STO93804/eu-ai-act-first-regulation-on-artificial-intelligence)). This could significantly impact how AI is utilized, not just in drug development but across various sectors.

Interestingly, while many startups were initially racing to emphasize the role of AI in their solutions, the new regulatory landscape might induce some to tone down their AI capabilities to avoid stringent scrutiny. It will be fascinating to see how this plays out in the pharmaceutical industry, where compliance and safety are paramount.

This adds another layer of complexity to adopting AI in drug development. Companies might have to tread more cautiously and possibly recalibrate their strategies to align with evolving regulations.

**Comment by Coutuan3s** [comment]
I'll just say this: Sanofi goes 'all in' on AI with new app to support company's manufacturing, R&D and more
By [Zoey Becker](https://www.fiercepharma.com/person/zoey-becker) Jun 14, 2023
